__tests__/app.test.js

```diff
@@ -156,3 +124,54 @@ describe("\nGET /api/reviews/:review_id/comments\n", () => {
       });
   });
 });
+describe("\nPOST /api/reviews/:review_id/comments\n", () => {
+  test("get 201 status and add new review comment to database and return new comment", () => {
+    const review_id = 2;
+    const newComment = {
+      review_id: review_id,
+      username: "dav3rid",
+      body: "This is exactly what I was saying to my cat last night.",
+    };
+    return request(app)
+      .post(`/api/reviews/${review_id}/comments`)
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        expect(response.body).toMatchObject({
+          body: expect.any(String),
```

Comment:
Make sure to test against what you know the comment object should look like. You provided the body, so hopefully it comes back the same